File: Curso-py/clase_6-9.py
def saludar(nom):
  return "Hola, " + nom + "!"
saludo_pers = saludar("Carlos")
print(saludo_pers)
edad = 20
if edad >= 18:
  print("Eres mayor de edad.")
else:
  print("Eres menor de edad.")
for i in range(5):
  print("Número:", i)

File: Curso-js/clase_6-9.js
function saludar(nom) {
  return "Hola, " + nom + "!";
}
var saludoPers = saludar("Maria");
console.log(saludoPers);
var edad = 18;
if (edad >= 18) {
  console.log("Eres mayor de edad.");
} else {
  console.log("Eres menor de edad.");
}
for (var i = 0; i < 5; i++) {
  console.log("Número:", i);
}